UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/RootBodyPartContainer.cs

```diff
@@ -304,6 +316,11 @@ public virtual void ImplantPeriodicUpdate()
 		public virtual void SubBodyPartRemoved(BodyPart implant)
 		{
 			RemoveSpritesNID(implant);
+			if(ContainsLimbs.Count == 0)
```

Comment:
wouldn't this have issues if you have multiple right arms for example?

Reply:
whoops, forget to change that. on my way to fix it.

Reply:
fixed.